### PR TITLE
Add error code for ScanOperationStates table

### PR DESCRIPTION
**Pull request by erdembayar**
Addresses: https://github.com/NuGet/Engineering/issues/5250
Adding 2 new columns to persist extra details about scan operation

### CODE REVIEW

**Review by drewgillies — DISMISSED** [review]

File: Directory.Packages.props [hunk 1]
```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <EnablePackageVersionOverride>false</EnablePackageVersionOverride>
-    <ServerCommonPackageVersion>2.117.0</ServerCommonPackageVersion>
+    <ServerCommonPackageVersion>2.120.0-dev-eryondon-extraValidation-9271086</ServerCommonPackageVersion>
```
Comment (joelverhagen) [inline Directory.Packages.props:5]: TODO switch to official build

**Review by joelverhagen — DISMISSED** [review]

File: tests/NuGet.Services.Validation.Orchestrator.Tests/SymbolValidationMessageHandlerFacts.cs [hunk 2]
```diff
@@ -298,6 +299,11 @@ public MessageWithCustomDeliveryCount(IBrokeredMessage inner, int deliveryCount)
             public Task CompleteAsync() => throw new NotImplementedException();
             public string GetBody() => _inner.GetBody();
             public TStream GetBody<TStream>() => _inner.GetBody<TStream>();
+
+            public Stream GetRawBody()
+            {
+                throw new NotImplementedException();
```
Comment (erdembayar) [inline tests/NuGet.Services.Validation.Orchestrator.Tests/SymbolValidationMessageHandlerFacts.cs:305]: @agr 
I can see this interface change was introduced in https://github.com/NuGet/ServerCommon/pull/423. Should we try to implement it or just leave `Not implemented` as it's here?

File: tests/NuGet.Services.Validation.Orchestrator.Tests/ValidationMessageHandlerFacts.cs [hunk 3]
```diff
@@ -301,6 +302,11 @@ public MessageWithCustomDeliveryCount(IBrokeredMessage inner, int deliveryCount)
             public Task CompleteAsync() => throw new NotImplementedException();
             public string GetBody() => _inner.GetBody();
             public TStream GetBody<TStream>() => _inner.GetBody<TStream>();
+
+            public Stream GetRawBody()
+            {
+                throw new NotImplementedException();
```
Comment (erdembayar) [inline tests/NuGet.Services.Validation.Orchestrator.Tests/ValidationMessageHandlerFacts.cs:308]: Same as above https://github.com/NuGet/NuGet.Jobs/pull/1187/files#r1537954170

**Review by joelverhagen — APPROVED** [review]
PR seems to have extra commits. Might need a rebase on `dev`.

File: Directory.Packages.props [hunk 4]
```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <EnablePackageVersionOverride>false</EnablePackageVersionOverride>
-    <ServerCommonPackageVersion>2.117.0</ServerCommonPackageVersion>
+    <ServerCommonPackageVersion>2.220.0-main-9301227</ServerCommonPackageVersion>
     <NuGetClientPackageVersion>6.9.1</NuGetClientPackageVersion>
-    <NuGetGalleryPackageVersion>4.4.5-dev-9113701</NuGetGalleryPackageVersion>
+    <NuGetGalleryPackageVersion>4.4.6-dev-eryondon-extraValidation-9293959</NuGetGalleryPackageVersion>
```
Comment (agr) [inline Directory.Packages.props:7]: This must be updated.

File: tests/NuGet.Services.Validation.Orchestrator.Tests/SymbolValidationMessageHandlerFacts.cs (hunk 2 above)
Comment (agr) [reply tests/NuGet.Services.Validation.Orchestrator.Tests/SymbolValidationMessageHandlerFacts.cs:305]: This is only used by monitoring jobs. It's fine to throw `NotImplementedException` here.

File: Directory.Packages.props (hunk 4 above)
Comment (erdembayar) [reply Directory.Packages.props:7]: Created PR to address it: https://github.com/NuGet/NuGet.Jobs/pull/1193

**Pull request merged**